Cannot find module 'eslint-plugin-jsx-a11y'

### pnpm version: 5.3.0

### Code to reproduce the issue:
I'm using vim with Conquer of Copletition to develop React application. And the ESLint tools is not finding the `eslint-plugin-jsx-a11y`

Not sure but this could be a eslint-plugin error, a CoC error. But using `npm` or `yarn` does not present this error.

### Expected behavior:

### Actual behavior:

### Additional information:

 - `node -v` prints: v14.4.0
 - Windows, OS X, or Linux?: Mac OS Catalina 10.15.5 (19F101)

```
[coc.nvim] Error on willSaveWaitUntil: Error: Request textDocument/willSaveWaitU
ntil failed with message: Failed to load plugin 'jsx-a11y' declared in '.eslintr
c.json » plugin:@wordpress/recommended » /Users/mario/Projects/wp-devenv/wp/wp-c
ontent/plugins/ihs-appointments/node_modules/.pnpm/@wordpress/eslint-plugin@7.1.
0_eslint@7.4.0/node_modules/@wordpress/eslint-plugin/configs/recommended-with-fo
rmatting.js » /Users/mario/Projects/wp-devenv/wp/wp-content/plugins/ihs-appointm
ents/node_modules/.pnpm/@wordpress/eslint-plugin@7.1.0_eslint@7.4.0/node_modules
/@wordpress/eslint-plugin/configs/jsx-a11y.js': Cannot find module 'eslint-plugi
n-jsx-a11y'
Require stack:
- /Users/mario/Projects/wp-devenv/wp/wp-content/plugins/ihs-appointments/__place
holder__.js
Referenced from: /Users/mario/Projects/wp-devenv/wp/wp-content/plugins/ihs-appoi
ntments/node_modules/.pnpm/@wordpress/eslint-plugin@7.1.0_eslint@7.4.0/node_modu
les/@wordpress/eslint-plugin/configs/jsx-a11y.js
```

![Screenshot](https://user-images.githubusercontent.com/3419025/86993588-fdae3200-c169-11ea-8cc8-963d9330c95b.png)

--- a/.changeset/nasty-teachers-matter.md
+++ b/.changeset/nasty-teachers-matter.md
@@ -1,0 +1,5 @@
+---
+"@pnpm/config": minor
+---
+
+Hoist all ESLint plugins to the root of node_modules by default.

--- a/packages/config/src/index.ts
+++ b/packages/config/src/index.ts
@@ -147,7 +147,7 @@ export default async (
     'link-workspace-packages': true,
     'package-lock': npmDefaults['package-lock'],
     'pending': false,
-    'public-hoist-pattern': ['@types/*'],
+    'public-hoist-pattern': ['@types/*', 'eslint-plugin-*'],
     'recursive-install': true,
     'registry': npmDefaults.registry,
     'save-peer': false,

